src/jobs/PkgEvalJob.jl

```diff
@@ -1114,14 +1121,14 @@ function printreport(io::IO, job::PkgEvalJob, results)
                     end
                     against_status = String(test.status_1)
                     print(io, "[$against_status]($against_log) | ")
-                    print(io, "$(get(history, test.package, "missing")) |")
+                    print(io, "<span class=\"history\">$(get(history, test.package, "missing"))</span> |")
```

Comment:
Good solution!

Reply:
Eh, it's not great, as it embeds HTML now in Markdown. I tried using the AttributeRule from CommonMark.jl, but that only allows attaching IDs to entities, not to plain text (so `*history*{{#id}}` would work, but then the history string would be emphasized). I couldn't find a way to create non-semantic group in Markdown, so and just ended up using HTML instead...